src/promptflow/tests/executor/e2etests/test_executor_validation.py

```diff
@@ -204,6 +204,19 @@ def test_flow_run_input_type_invalid(self, flow_folder, line_input, error_class,
         with pytest.raises(error_class):
             executor.exec_line(line_input)
 
+    def test_flow_run_input_assignment(self, dev_connections):
+        # Flow run -  the input is from get_partial_line_inputs()
+        flow_folder = "simple_flow_with_python_tool"
+        executor = FlowExecutor.create(get_yaml_file(flow_folder, FLOW_ROOT), dev_connections, raise_ex=False)
+        invalid_input = {"num": "hello"}
```

Comment:
If invalid inputs not saved to run info, will it reduce debuggability? Usually, user want to view the invalid inputs to see what happened. Right?

Reply:
Or we could encourage customer to debug by error response and local input data?